### PR TITLE
API-3875

### PR DESCRIPTION
**Pull request by cjrowe**

### CODE REVIEW

File: app/uk/gov/hmrc/thirdpartyapplication/services/ApplicationService.scala [hunk 1]
```diff
@@ -137,7 +137,7 @@ class ApplicationService @Inject()(applicationRepository: ApplicationRepository,
     fetchApp(applicationId) flatMap { app =>
 
       def updateWso2Subscriptions(): Future[Seq[HasSucceeded]] = {
-        apiGatewayStore.getSubscriptions(app.wso2Username, app.wso2Password, app.wso2ApplicationName) flatMap { originalApis =>
+        subscriptionRepository.getSubscriptions(applicationId) flatMap { originalApis =>
```
Comment (lburgos) [inline app/uk/gov/hmrc/thirdpartyapplication/services/ApplicationService.scala]: I don't think we need this change, as this entire method (updateWso2Subscriptions) will be removed once we switch over to AWS (updating an app to another tier won't require to update the subscriptions).

**Comment by lburgos** [comment]
Somewhat related to this PR, it may be a good idea to delete these 2 that are not needed since we changed the way we deal with subscriptions in AWS:

https://github.com/hmrc/third-party-application/blob/master/app/uk/gov/hmrc/thirdpartyapplication/services/ApiGatewayStore.scala#L146

and 

https://github.com/hmrc/third-party-application/blob/master/app/uk/gov/hmrc/thirdpartyapplication/services/ApiGatewayStore.scala#L161

File: app/uk/gov/hmrc/thirdpartyapplication/services/ApplicationService.scala (hunk 1 above)
Comment (cjrowe) [reply app/uk/gov/hmrc/thirdpartyapplication/services/ApplicationService.scala]: Have reverted that now

**Comment by cjrowe** [comment]
> Somewhat related to this PR, it may be a good idea to delete these 2 that are not needed since we changed the way we deal with subscriptions in AWS:
> 
> https://github.com/hmrc/third-party-application/blob/master/app/uk/gov/hmrc/thirdpartyapplication/services/ApiGatewayStore.scala#L146
> 
> and
> 
> https://github.com/hmrc/third-party-application/blob/master/app/uk/gov/hmrc/thirdpartyapplication/services/ApiGatewayStore.scala#L161

Removed those references now...

**Pull request merged**